var/spack/repos/builtin/packages/exodusii/package.py

```diff
@@ -15,15 +17,52 @@ class Exodusii(CMakePackage):
     (problem definition), postprocessing (results visualization), and
     data transfer between codes.  An Exodus II data file is a random
     access, machine independent, binary file that is written and read
-    via C, C++, or Fortran API routines.
+    via C, C++, or Fortran API routines.  This package *only* installs
+    the C and optionally Fortran library for exodus.  If you want the full
+    suite of exodus-releated tools including the IOSS library, install
+    the seacas package instead of this package.
     """
 
-    homepage = "https://github.com/gsjaardema/seacas"
-    git = "https://github.com/gsjaardema/seacas.git"
-    url = "https://github.com/gsjaardema/seacas/archive/refs/tags/v2021-04-05.zip"
+    homepage = "https://sandialabs.github.io/seacas/"
+    git = "https://github.com/sandialabs/seacas.git"
+    url = "https://github.com/sandialabs/seacas/archive/v2019-08-20.tar.gz"
+    maintainers("gsjaardema")
 
-    license("X11")
+    license("BSD-3-Clause")
 
+    version("master", branch="master")
+    version(
+        "2024-03-11", sha256="b849d958b34e77300aaf331f29c3e6fe417fd82600850a82e674a9b7ba4045ff"
+    )
+    version(
+        "2023-11-27", sha256="fea1c0a6959d46af7478c9c16aac64e76c6dc358da38e2fe8793c15c1cffa8fc"
+    )
+    version(
+        "2023-10-24",
+        sha256="f93bf0327329c302ed3feb6adf2e3968f01ec325084a457b2c2dbbf6c4f751a2",
+        deprecated=True,
+    )
+    version(
+        "2023-05-30", sha256="3dd982841854466820a3902163ad1cf1b3fbab65ed7542456d328f2d1a5373c1"
+    )
+    version(
+        "2022-10-14", sha256="cde91e7561d2352045d669a25bdf46a604d85ed1ea7f3f5028004455e4ce9d56"
+    )
+    version(
+        "2022-05-16", sha256="22ff67045d730a2c7d5394c9034e44a2033cc82a461574f93d899e9aa713d4ae"
+    )
+    version(
+        "2022-03-04", sha256="a934a473e1fdfbc8dbb55058358551a02e03a60e5cdbf2b28b8ecd3d9500bfa5"
+    )
+    version(
+        "2022-02-16", sha256="a6accb9924f0f357f63a01485c3eaaf5ceb6a22dfda73fc9bfb17d7e2f565098"
+    )
+    version(
+        "2022-01-27", sha256="beff12583814dcaf75cf8f1a78bb183c1dcc8937bc18d5206672e3a692db05e0"
+    )
+    version(
+        "2021-10-11", sha256="f8a6dac813c0937fed4a5377123aa61d47eb459ba87ddf368d02ebe10c2c3a0d"
+    )
     version(
         "2021-04-05", sha256="f40d318674753287b8b28d2b4e5cca872cd772d4c7383af4a8f3eeb48fcc7ec0"
```

Comment:
These checksums are no longer valid. 

You could add `url_for_version` method (see https://spack.readthedocs.io/en/latest/packaging_guide.html#writing-a-package-recipe) or add `url` as an argument to the directory.

Reply:
I changed back to using the url that fetches zip files instead of tar.gz files and redid all the new version checksums.  I installed a few of the older versions and new versions and I think all checksums are valid.